internal/policy/policy.go

```diff
@@ -56,11 +56,11 @@ var ErrPolicyExists = errors.New("cannot initialize Policy namespace as it exist
 // has a zero hash.
 func InitializeNamespace(repo *git.Repository) error {
 	for _, name := range []string{PolicyRef /*, PolicyStagingRef*/} {
-		if _, err := repo.Reference(plumbing.ReferenceName(name), true); err != nil {
+		if policy, err := repo.Reference(plumbing.ReferenceName(name), true); err != nil {
```

Comment:
```suggestion
		if ref, err := repo.Reference(plumbing.ReferenceName(name), true); err != nil {
```

nit: let's not call it `policy`